README.md

```diff
@@ -89,6 +89,13 @@ The script creates two output files, in the file names specified by the `details
 - MER (match error rate) is the proportion of I/O word matches which are errors.
 - WIL (word information lost) is a simple approximation to the proportion of word information lost which overcomes the problems associated with the RIL (relative information lost) measure that was proposed half a century ago.
 
+## Execution - Optional Grid Execution
+Use the experiment.py script to execute a series of experients where configuration settings may change for each experiment.  This option may require customization for each specific situation.
```

Comment:
Let's brand it as a sample - it definitely requires customization to grid search different parameters.

Reply:
Changed documentation to Experiments and not Optional.